### ISSUE TITLE
NPE in importData

### ISSUE THREAD

**Issue by stevensegers** [DeBortoliWines/openerp-java-api#24]
When Odoo returns an error that is not linked to a specific field, the importData functions throws NPE while trying to parse the return message.
```
2015/11/28 16:52:09 - Insert Timesheet.0 - Caused by: java.lang.NullPointerException
2015/11/28 16:52:09 - Insert Timesheet.0 - 	at com.debortoliwines.openerp.api.ObjectAdapter.importData(ObjectAdapter.java:532)
2015/11/28 16:52:09 - Insert Timesheet.0 - 	at org.pentaho.di.trans.steps.openerp.objectoutput.OpenERPObjectOutput.CommitBatch(OpenERPObjectOutput.java:175)
2015/11/28 16:52:09 - Insert Timesheet.0 - 	at org.pentaho.di.trans.steps.openerp.objectoutput.OpenERPObjectOutput.processRow(OpenERPObjectOutput.java:68)
2015/11/28 16:52:09 - Insert Timesheet.0 - 	... 2 more
```

**Comment by stevensegers** [comment]
I created a pull request for a potential fix: https://github.com/DeBortoliWines/openerp-java-api/pull/25

**Comment by fpoyer** [comment]
This issue will be fixed by PR #28, please close it when merging the PR.

**Comment by flotho** [comment]
done!
